application.go

```diff
@@ -480,7 +478,6 @@ func (r *marathonClient) DeleteApplication(name string) (*DeploymentID, error) {
 // RestartApplication performs a rolling restart of marathon application
 // 		name: 		the id used to identify the application
 func (r *marathonClient) RestartApplication(name string, force bool) (*DeploymentID, error) {
-	glog.V(debugLevel).Infof("restarting the application: %s, force: %s", name, force)
```

Comment:
Right now we log `RestartApplication`, `ScaleApplicationInstances` and `UpdateApplication` while there is no log for `DeleteApplication`, `CreateApplication`, `SetApplicationVersion` and all the other "helper" funcs in `applications.go`. 

I'd rather have no logs (and let the user decide what to log) than inconsistent logs. What happens is, people start relying on logs, and then get confused if one expected log is missing. Like `DeleteApplication` for example. (People in this context could be people not familiar with the code, like devops)

I also think that logs here would just generate noise. The func's input and output variables hold all state and the user is always able to receive that state without missing information. This is different for `httpRequest` for example where we log raw http response, which is not returned to the user.